.NET/Microsoft.Recognizers.Text.DateTime/Dutch/Parsers/DutchHolidayParserConfiguration.cs

```diff
@@ -1,100 +1,101 @@
-                { "youthday", YouthDay },
-                { "childrenday", ChildrenDay },
-                { "femaleday", FemaleDay },
-                { "treeplantingday", TreePlantDay },
-                { "arborday", TreePlantDay },
-                { "girlsday", GirlsDay },
-                { "whiteloverday", WhiteLoverDay },
-                { "loverday", ValentinesDay },
-                { "christmas", ChristmasDay },
-                { "xmas", ChristmasDay },
-                { "newyear", NewYear },
-                { "newyearday", NewYear },
-                { "newyearsday", NewYear },
-                { "inaugurationday", InaugurationDay },
-                { "groundhougday", GroundhogDay },
-                { "valentinesday", ValentinesDay },
-                { "stpatrickday", StPatrickDay },
-                { "aprilfools", FoolDay },
-                { "stgeorgeday", StGeorgeDay },
-                { "mayday", Mayday },
-                { "cincodemayoday", CincoDeMayoday },
-                { "baptisteday", BaptisteDay },
-                { "usindependenceday", UsaIndependenceDay },
-                { "independenceday", UsaIndependenceDay },
-                { "bastilleday", BastilleDay },
-                { "halloweenday", HalloweenDay },
-                { "allhallowday", AllHallowDay },
-                { "allsoulsday", AllSoulsday },
-                { "guyfawkesday", GuyFawkesDay },
-                { "veteransday", DutchVeteransday },
-                { "christmaseve", ChristmasEve },
-                { "newyeareve", NewYearEve },
-                { "easterday", EasterDay },
-                { "kingsday", KingsDay },
-                { "queensday", QueensDay },
-                { "prinsjesdag", Prinsjesdag },
-                { "eerstekerstdag", ChristmasDay },
-                { "tweedekerstdag", SecondChristmasDay },
-                { "bevrijdingsdag", Bevrijdingsdag },
-                { "dodenherdenking", Dodenherdenking },
-                { "dagvandearbeid", Dagvandearbeid },
-            };
-        }
-
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using Microsoft.Recognizers.Definitions.Dutch;
+using DateObject = System.DateTime;
+
+namespace Microsoft.Recognizers.Text.DateTime.Dutch
+{
+    public class DutchHolidayParserConfiguration : BaseHolidayParserConfiguration
+    {
+        public DutchHolidayParserConfiguration(IDateTimeOptionsConfiguration config)
+            : base(config)
+        {
+            this.HolidayRegexList = DutchHolidayExtractorConfiguration.HolidayRegexList;
+            this.HolidayNames = DateTimeDefinitions.HolidayNames.ToImmutableDictionary();
+        }
+
+        public override int GetSwiftYear(string text)
+        {
+            var trimmedText = text.Trim().ToLowerInvariant();
+            var swift = -10;
+            if (trimmedText.StartsWith("volgend") || trimmedText.StartsWith("volgende"))

```

Comment:
Such constants should be hard-coded. Can you move them to YAML?
There are other examples of similar cases in the English YAML that could be used as a base.

Reply:
values already in the yaml. changed in the parser to check from the yaml.

Reply:
These strings ("volgend" and "volgende") should not be hardcoded here. Their values need to be read from the YAML file (similarly to DateTimeDefinitions.HolidayNames). English has examples of such cases.